apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/PageController.java

```diff
@@ -128,4 +142,14 @@ public PageStateResponse deletePageDraft(@PathVariable("id") Long pageId) {
     }
 
 
+    @GetMapping("downloadPageMetadata/{waTemplateUid}")
```

Comment:
suggestion: Other page APIs define their routes as `api/v1/pages/{id}`

Thoughts on making the metadata route: `{id}/metadata` (the `api/v1/pages` piece is declared at class level)